pandas/core/frame.py

```diff
@@ -6440,6 +6440,8 @@ def append(self, other, ignore_index=False,
     def join(self, other, on=None, how='left', lsuffix='', rsuffix='',
              sort=False):
         """
+        Append columns of another DataFrame.
```

Comment:
Can you replace `append` by `join`? I think in general they mean the same, but in this context may give the idea that the they are being added at the end of the DataFrame, or that they are not aligned.